ceph-releases/jewel/ubuntu/14.04/demo/entrypoint.sh

```diff
@@ -78,6 +78,14 @@ public network = ${CEPH_PUBLIC_NETWORK}
 cluster network = ${CEPH_PUBLIC_NETWORK}
 ENDHERE
 
+		# For ext4
```

Comment:
@leseb Because the `if` for checking ext4 is under `if [ ! -e /etc/ceph/${CLUSTER}.conf ]`, so it has 2 tabs.

If the content needs to be indented too, I should use `<<-ENDHERE` instead of `<<ENDHERE`.

Do you want me to do so?